common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -406,7 +406,7 @@ div.problem {
       margin-top: 3px;
 
       .MathJax_Display {
-        display: inline-block;
+        display: inline-block !important;
```

Comment:
Somewhere else, that CSS entry is marked as !important already. This is much more specific, and needs to override it.

That other location is from this file, I think. It hasn't been modified in years, but I don't know where else that important is actually important.
edx-platform/common/static/js/vendor/mathjax-MathJax-c9db6ac/docs/source/mjtheme/static/